packagedb/models.py

```diff
@@ -456,11 +456,12 @@ class PackageContentType(models.IntegerChoices):
     # fields matches with the current package
     CURATION = 1, 'curation'
     PATCH = 2, 'patch'
-    SOURCE_REPO = 3, 'source_repo'
-    SOURCE_ARCHIVE = 4, 'source_archive'
-    BINARY = 5, 'binary'
-    TEST = 6, 'test'
-    DOC = 7, 'doc'
+    METADATA = 3, 'metadata'
```

Comment:
I think we need a migration for existing packages that have `package_content` set, since SOURCE_ARCHIVE is now 5 instead of 4 and BINARY is 6 instead of 5